### DISCUSSION TITLE
OSC EDIT MOTION parameter (in STEP EDIT) doesn't work

### DISCUSSION THREAD

**Discussion by bukru** [bangcorrupt/hacktribe#310] [Q&A] [answered]
Hi all

When in STEP EDIT mode, the parameter OSC EDIT MOTION does nothing.

Does anybody has the same problem ?

**Comment by bangcorrupt** [comment]
I haven't tried it for a while, but I think it only works if you record a motion sequence first.

**Reply by bukru** [reply]
I tried juste now, it does not work.

**Reply by BKLronin** [reply]
Works for me when the track has a motion recorded. Can be deleted but needs to be done once per track I think.

**Reply by bukru** [reply]
When you say a motion you talk about OSC motion or an other ?

**Reply by bukru** [reply, marked as answer]
Ok so i tried again and i need to record an OSC EDIT automation before if work.
Thanks for informations guys.
